integrated_channels/canvas/views.py

```diff
@@ -0,0 +1,112 @@
+"""
+Views containing APIs for cornerstone integrated channel
+"""
+
+from django.conf import settings
+from django.apps import apps
+import requests
+from rest_framework.response import Response
+from rest_framework import generics, status
+from six.moves.urllib.parse import urljoin
+
+from enterprise.utils import get_enterprise_customer
+from integrated_channels.canvas.models import CanvasEnterpriseCustomerConfiguration
+class CanvasCompleteOAuthView(generics.ListAPIView):
```

Comment:
Think you will need two lines here to pass pylint. I typically run `pylint --rcfile=pylintrc integrated_channels/canvas/` or sth like it in lms shell. or use the vscode python run lint thingie (make sure to choose pylint as your linter)

Reply:
The commands actually being run are in the tox.ini file. I just ran them manually a few minutes ago as well:
```
pylint -j 0 enterprise enterprise_learner_portal consent integrated_channels tests test_utils requirements/check_pins.py
pycodestyle enterprise enterprise_learner_portal consent integrated_channels tests test_utils
pydocstyle enterprise enterprise_learner_portal consent integrated_channels tests test_utils
isort --check-only --diff --recursive tests test_utils enterprise enterprise_learner_portal consent integrated_channels manage.py setup.py
```

You can also just run these from a minimally setup virtualenv if you - like me - do not have edx-enterprise checked out into src and don't feel like waiting 5 million years for that cp command to finish.

Reply:
Thanks to @raq929, I was advised to put enterprise in src from early on  👀

Reply:
I could always do so at any time, I don't know why I'm dying on this hill.